packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -26,9 +26,12 @@ spec:
           volumeMounts:
             - mountPath: /fluent-bit/etc/
               name: config
-            - mountPath: /run/log/journal
+            - mountPath: /var/log/journal
```

Comment:
This path doesn't exist when I look on centos 7, centos 8, and opensuse. It only exists on ubuntu.